src/components/ReportActionItem/TripDetailsView.tsx

```diff
@@ -142,10 +134,22 @@ function ReservationView({reservation, transactionID, tripRoomReportID, reservat
     );
 }
 
+type TripDetailsViewProps = {
+    /** The active tripRoomReportID, used for Onyx subscription */
+    tripRoomReportID?: string;
+
+    /** Whether we should display the horizontal rule below the component */
+    shouldShowHorizontalRule: boolean;
+};
+
 function TripDetailsView({tripRoomReportID, shouldShowHorizontalRule}: TripDetailsViewProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
 
+    if (!tripRoomReportID) {
+        return null;
+    }
+
     const tripTransactions = ReportUtils.getTripTransactions(tripRoomReportID);
```

Comment:
I think this is a bad function because it breaks the reactivity of this component. It should get the transactions with `useOnyx()`. I'm going to open a GH to remove it.

Reply:
GH is here: https://github.com/Expensify/App/issues/55077